dev/ansible/00_create_intention.yml

```diff
@@ -0,0 +1,29 @@
+- name: set variable
+  set_fact:
+    consul_token: "{{ lookup('env', 'consul_master_token') }}"
+
+- name: Check if intention nifi => nifi_registry (allow) exists
+  shell: consul intention match -source nifi
+  register: nifi_intention
+  when: lookup('env', 'consul_acl') | bool and lookup('env', 'consul_acl_default_policy') == 'deny'
+  environment:
+    CONSUL_HTTP_TOKEN: "{{ consul_token }}"
```

Comment:
You have wrong consul_token, it must be `'consul_master_token'`. I suggest that you include env `CONSUL_HTTP_ADDR` such that:
```
environment:
    CONSUL_HTTP_ADDR: "{{ lookup('env', 'CONSUL_HTTP_ADDR') }}"
    CONSUL_HTTP_TOKEN: "{{ lookup('env', 'consul_master_token') }}"
```

Reply:
I have included consul_http_addr , but if you check line 3 consul_token: "{{ lookup('env', 'consul_master_token') }}", then this line number 3 works as same   for  line 10 .
CONSUL_HTTP_TOKEN: "{{ consul_token }}" means  consul_token: "{{ lookup('env', 'consul_master_token') }}"